docs/UI/CancelInteraction/assets/CancelInteractionAnyAlert.puml

```diff
@@ -0,0 +1,17 @@
+@startuml
+
+participant Mobile
+participant Core
+participant HMI
+
+Mobile -> Core : CancelInteractionRequest\n(appID, ALERT)
```

Comment:
Should probably include the `Alert` message in this flow as well.

Reply:
Another small note, I believe in this context `ALERT` should be replaced with `AlertID`

Reply:
AlertID used in proper places, added Alert to the flow for all diagrams in https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/163/commits/1f93da0ff41d60f00c4d1bb3cc93ffeb5bd034e0